docs/build/custom.md

```diff
@@ -113,80 +113,122 @@ The `MinSizeRel` configuration will produce the smallest binary size.
 
 The `Release` configuration can also be used if you wish to prioritize performance over binary size.
 
-## Custom build package format
+## Version of ONNX Runtime to build from
+
+Unless there is a specific feature you need, do not use the unreleased `master` branch.
+
+Once you have cloned the ONNX Runtime repo, check out one of the release tags to build from.
+
+```bash
+git clone --recursive https://github.com/microsoft/onnxruntime
+cd onnxruntime
+git checkout <release tag>
+```
+
+Release tag names follow the pattern `v<release version>`. For example, `v1.11.0`.
+Find them [here](https://github.com/microsoft/onnxruntime/tags).
+
+## Example build commands
+
+### Build on windows, with reduced operator support, and support for ORT format models only
+
+`<ONNX Runtime repository root>.\build.bat --config=MinSizeRel --cmake_generator="Visual Studio 16 2019" --build_shared_lib --minimal_build --disable_ml_ops --disable_exceptions --include_ops_by_config <config file from model conversion> --skip_tests`
+
```

Comment:
nit: should we include `--disable_rtti` and `--enable_reduced_operator_type_support` here?